gazelle/rules_python_gazelle_helper/__main__.py

```diff
@@ -16,10 +16,16 @@
 # STDIN receives parse requests, one per line. It outputs the parsed modules and
 # comments from all the files from each request.
 
-import parse
-import std_modules
 import sys
 
+# NOTE @aignas 2023-12-02: Use absolute imports with respect to WORKSPACE root.
```

Comment:
It would be better to not rely on the repo root being added to sys.path -- that's a behavior I plan to disable once the Starlark impl can be enabled.

But, I can understand this is just a bugfix CL, so this is fine for now. Is Gazelle able to under PyInfo.imports, or have some equivalent?

Reply:
This made me realize that I just needed to use `imports = ["."]` on the `py_binary` that is being used here to fix the error and no moving of files is actually needed. Thanks!